src/chrome/content/rules/localmonero.co.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="localmonero.co">
+	<target host="localmonero.co" />
+	<target host="meet.localmonero.co" />
+  <target host="forums.localmonero.co" />
+
+	<rule from="^http:" to="https:" />
+</ruleset>
```

Comment:
```suggestion
<ruleset name="localmonero.co">
	<target host="localmonero.co" />
	<target host="www.localmonero.co" />
	<target host="meet.localmonero.co" />
	<target host="mta-sts.agoradesk.com" />
	<target host="forums.localmonero.co" />

	<rule from="^http:" to="https:" />
</ruleset>
```

Reply:
You seem to have accidentally added "mta-sts.agoradesk.com" to the LocalMonero rule.

Reply:
@LocalMonero Thanks for noticing!

Reply:
mta-sts.localmonero.co isn't really needed, though.

Reply:
@LocalMonero mta-sts.localmonero.co is removed.

Reply:
@LocalMonero Are these changes OK? If yes, please press "Commit suggestion", and after that I will request review from another maintainer (I can't merge PRs by myself).

Reply:
@pipboy96 Done.